render.js

```diff
@@ -5,19 +5,19 @@ const Renderer = function(){
         $("#posts").empty()
         for(let i in posts){
             let id = posts[i].id
-            const div = $(`<div class="post" data-postId="${id}"><div>`)
+            const div = $(`<div class="post" data-postId="${id}"></div>`)
             // console.log(id)
-            div.append(`<div class="post" data-postId="${id}"> ${posts[i].text} <div>`)
-            div.append(`<div class="delete" data-postId="${id}">delete post<div>`)
+            div.append(`<div class="post" data-postId="${id}"> ${posts[i].text} </div>`)
```

Comment:
save each of these as variables. When you put something so big inside the `.append()` method it makes the code difficult to follow. The best code is the most simple and most clean. Adding more variables can help.

`const newPostId = post.id`
`const postBox = $("<div class='pcon' data-id="+ ${newPostId} + "> <h1>" + post.text + "</h1></div>")`
`$("#posts").append(postBox)`

Reply:
i'll try keeping things more simple and clean
thank you for your review.